features/step_definitions/basic_steps.rb

```diff
@@ -8,3 +8,6 @@
     end
 end
 
+When("I click {string}") do |string|
```

Comment:
Use a variable name that is more descriptive of the content

Reply:
done